index.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/extensions */
```

Comment:
- [ ] Kindly remove all of the `eslint-disable` commands from your project. It's fine to only use it inside the Luxon module since that is actually a external dependency but for the rest of the project, linters are there for best practice so using disable is never recommended.